hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/WritableNftStoreTest.java

```diff
@@ -150,7 +164,7 @@ void removesByTokenIdAndSerialNum() {
         writableNftStore = new WritableNftStore(writableStates);
         assertNotNull(writableNftStore.get(nftToRemove));
 
-        writableNftStore.remove(asToken(nftToRemove.tokenTypeNumber()), nftToRemove.serialNumber());
+        writableNftStore.remove(asToken(nftToRemove.tokenId().tokenNum()), nftToRemove.serialNumber());
```

Comment:
Count this be simplified?
`writableNftStore.remove(nftToRemove.tokenId(), nftToRemove.serialNumber());`

Reply:
Updated, thanks.